pentasharp/Controllers/BusScheduleCompanyController.cs

```diff
@@ -1,15 +1,242 @@
+using AutoMapper;
 using Microsoft.AspNetCore.Mvc;
+
+using Microsoft.EntityFrameworkCore;
+using pentasharp.Data;
+using pentasharp.Mappings;
+using pentasharp.Models.Entities;
+using pentasharp.ViewModel.Bus;
+using System;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Runtime.Intrinsics.X86;
+
 using Microsoft.AspNetCore.Mvc.Filters;
 using WebApplication1.Filters;
 
-namespace WebApplication1.Controllers
+
+
+namespace pentasharp.Controllers
 {
     [ServiceFilter(typeof(AdminOnlyFilter))]
     public class BusScheduleCompanyController : Controller
```

Comment:
how are you testing such features, I believe you are not using postman since I dont see any paths on the actions below. If pentasharp doesnt have a collection on postman yet, please create one. 
You should also add routes to each endpoint, For example: [Route("api/bus/edit")]
Having everything in postman is a good practice, specifically when we want to ensure our endpoint works as it should in isolation from frontend

Reply:
can i have a reply on the above?